press/api/client.py

```diff
@@ -4,12 +4,38 @@
 from __future__ import unicode_literals
 import frappe
 import inspect
+from frappe.utils import cstr
 from pypika.queries import QueryBuilder
 from frappe.model.base_document import get_controller
 from frappe.model import default_fields, child_table_fields
 from frappe import is_whitelisted
 from frappe.handler import get_attr, run_doc_method as _run_doc_method
```

Comment:
If we are allowing run_doc_method then it will start allowing all currently whitelisted functions and a lot of them are whitelisted for Frappe team via Desk. Either we need to audit them all or start a new list of whitelisted methods specifically for FC end users like `@press.whitelist()`